monodoc/Mono.Utilities/colorizer.cs

```diff
@@ -1,30 +1,27 @@
 using System;
 using System.Text.RegularExpressions;
 using System.Collections;
-
-namespace Mono.Utilities {
-	public class Colorizer {
-		//
-		// Syntax coloring
-		//
-
-		static string keywords_cs =
-			"(\\babstract\\b|\\bevent\\b|\\bnew\\b|\\bstruct\\b|\\bas\\b|\\bexplicit\\b|\\bnull\\b|\\bswitch\\b|\\bbase\\b|\\bextern\\b|"
-			+
-			"\\bobject\\b|\\bthis\\b|\\bbool\\b|\\bfalse\\b|\\boperator\\b|\\bthrow\\b|\\bbreak\\b|\\bfinally\\b|\\bout\\b|\\btrue\\b|"
-			+
-			"\\bbyte\\b|\\bfixed\\b|\\boverride\\b|\\btry\\b|\\bcase\\b|\\bfloat\\b|\\bparams\\b|\\btypeof\\b|\\bcatch\\b|\\bfor\\b|"
-			+
-			"\\bprivate\\b|\\buint\\b|\\bchar\\b|\\bforeach\\b|\\bprotected\\b|\\bulong\\b|\\bchecked\\b|\\bgoto\\b|\\bpublic\\b|"
-			+
-			"\\bunchecked\\b|\\bclass\\b|\\bif\\b|\\breadonly\\b|\\bunsafe\\b|\\bconst\\b|\\bimplicit\\b|\\bref\\b|\\bushort\\b|"
-			+
-			"\\bcontinue\\b|\\bin\\b|\\breturn\\b|\\busing\\b|\\bdecimal\\b|\\bint\\b|\\bsbyte\\b|\\bvirtual\\b|\\bdefault\\b|"
-			+
-			"\\binterface\\b|\\bsealed\\b|\\bvolatile\\b|\\bdelegate\\b|\\binternal\\b|\\bshort\\b|\\bvoid\\b|\\bdo\\b|\\bis\\b|"
-			+
-			"\\bsizeof\\b|\\bwhile\\b|\\bdouble\\b|\\block\\b|\\bstackalloc\\b|\\belse\\b|\\blong\\b|\\bstatic\\b|\\benum\\b|"
-			+ "\\bnamespace\\b|\\bstring\\b)";
+using System.Collections.Generic;
+using System.Linq;
+
+namespace Mono.Utilities
+{
+    public class Colorizer
+    {
+        //
+        // Syntax coloring
+        //
+        static string[] words = { "abstract","event","new","struct","as","explicit","null","switch","base","extern",
+                                                        "object","this","bool","false","operator","throw","break","finally","out","true",
```

Comment:
is this the correct format? looks like the indentation of second line is not correct.  try ctrl +k and ctrl + F